edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.IoTHub/EdgeAgentConnection.cs

```diff
@@ -128,7 +128,29 @@ public async Task UpdateReportedPropertiesAsync(TwinCollection patch)
             }
         }
 
-        public async Task SendEventBatchAsync(IEnumerable<Message> messages)
+        //// public async Task SendEventBatchAsync(IEnumerable<Message> messages)
```

Comment:
Can you remove all the old code instead of commenting it out? It's in the commit history...

Reply:
I'll remove this line, I'm going to leave the boilerplate in the connection handler since we did the same for device streams.

Reply:
There's commented-out code across 6 source files...do those files comprise the connection handler?